app/styles/app/modules/request/buttons.scss

```diff
@@ -1,3 +1,11 @@
+.repo-config .request-configs .request-configs-buttons {
+  top: -45px;
```

Comment:
The way that I would normally approach this situation would be to do something like 
```
.repo-config .request-configs .request-configs-buttons {
  position: relative;
  text-align: right;
```

This way, the changes in this PR to `app/styles/app/modules/request/configs.scss` become unnecessary. The way that it's currently set up, it essentially emulates relative positioning anyway, so I think it's more straightforward to just use relative position.

Reply:
i'll try that out. usually i first try things without absolute positioning. i have forgotten what the issue was that made me think i'd need to use it here.

Reply:
right ... i think the issue is that the `nav` element that holds the two tabs on the left side has `display: block`, so with `position: relative` on the next element that element is going to be pushed down to the next "line". 

![image](https://user-images.githubusercontent.com/2208/77368184-fe0df300-6d5b-11ea-9db8-090d65701d58.png)

even if i then move it back up using `top: -45px` that will then leave a gap:

![image](https://user-images.githubusercontent.com/2208/77368297-34e40900-6d5c-11ea-875d-4f9a5701d081.png)

unless we want to change the style of the tabs nav element (which i'm not sure would be preferable) i think the only way to fix this is to position the buttons absolutely, isn't it?

Reply:
Sorry, I think I didn't explain what I was proposing well enough :) Yes that's absolutely an appropriate use of absolute position, and I didn't intend to suggest getting rid of it entirely (though now I totally see how that came across). What I meant was to override it in the `.repo-config` context, which doesn't have those tabs. You're already using the selector with that class at the front to override the `top` declaration, and what I'm proposing is to instead use it to override the `position` declaration. So the top of this file would look like this:

```
.repo-config .request-configs .request-configs-buttons {
  position: relative;
  text-align: right;

  .request-configs-button-cancel {
    display: none;
  }
}

.request-configs .request-configs-buttons {
  position: absolute;
  top: 1px;
  right: 0;
}
```
With the `.request-configs .request-configs-buttons` styles being left alone. And then in `app/styles/app/modules/request/configs.scss` remove
```
.repo-config {
  position: relative;
  margin-top: 3em;
}
```
Does it make sense?

Reply:
oh, ok, i see now! yes, i'll give that a show.

Reply:
done. see here: https://github.com/travis-ci/travis-web/pull/2407/commits/ae78866c739f8786ee0b0eb357abe4bbef1122cb

Reply:
Looks good, thanks! :rocket: